.gitlab-ci.yml

```diff
@@ -142,15 +147,15 @@ before_script:
 # run tests for deb-x64
 run_tests_deb-x64:
   stage: source_test
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/deb_x64:latest
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/deb_x64:v955769-3225a59
```

Comment:
We have to check with the #engtools or #deveff guys if they ever do cleanup of tags other than `latest`, and if so what the criteria is.